erpnext/accounts/doctype/budget/test_budget.py

```diff
@@ -352,7 +352,10 @@ def test_monthly_budget_against_main_cost_center(self):
 
 def set_total_expense_zero(posting_date, budget_against_field=None, budget_against_CC=None):
 	if budget_against_field == "project":
-		budget_against = frappe.db.get_value("Project", {"project_name": "_Test Project"})
+		project = frappe.get_doc("Project", {"project_name": "_Test Project"})
```

Comment:
If the project `_Test Project` is created with `_Test Company` and the company is changed somewhere else, you may identify that spot and change it back to `_Test Company` post that assertion.

Changing it here may affect the reader's context.